conf/evolutions/bakery_db/1.sql

```diff
@@ -13,6 +13,36 @@ CREATE TABLE IF NOT EXISTS Product
     CONSTRAINT "bakery-inventory_pkey" PRIMARY KEY (id)
 );
 
+CREATE OR REPLACE FUNCTION update_modified_column()
+RETURNS TRIGGER AS $$
+BEGIN
+    NEW.updated_at = now();;
+    RETURN NEW;;
+END;;
+$$ language 'plpgsql';
```

Comment:
do you need this?

Reply:
The `language 'plpgsql';` part? I'm not sure, I put it as a sanity check.

Reply:
If you are not sure how this is used, please remove it. If the code doesn't work without it, please state in a comment or link in a comment as to why it was placed there

Reply:
I searched about it to understand more, I noticed that I am using a functions that requires plpgsql and removing it would break the program as it doesn't understand how to resolve some of the evolutionary file commands. I'll make sure to add a comment in as to what I wrote.